spec/features/work_packages/bulk/update_work_package_spec.rb

```diff
@@ -204,6 +204,101 @@
         end
       end
 
+      describe 'unsetting values for different fields' do
+        let(:boolean_cf) do
+          create(:boolean_wp_custom_field,
+                 name: 'Bool CF',
+                 types: [type],
+                 projects: [project])
+        end
+        let(:required_boolean_cf) do
+          create(:boolean_wp_custom_field,
+                 name: 'Required Bool CF',
+                 types: [type],
+                 projects: [project],
+                 is_required: true)
+        end
+        let(:list_cf) do
+          create(:list_wp_custom_field,
+                 name: 'List CF',
+                 types: [type],
+                 projects: [project],
+                 possible_values: %w[A B C])
+        end
+        let(:required_list_cf) do
+          create(:list_wp_custom_field,
+                 name: 'Required List CF',
+                 types: [type],
+                 projects: [project],
+                 possible_values: %w[A B C],
+                 is_required: true)
+        end
+        let(:multi_list_cf) do
+          create(:list_wp_custom_field, :multi_list,
+                 name: 'Multi select List CF',
+                 types: [type],
+                 projects: [project],
+                 possible_values: %w[A B C])
+        end
+        let(:user_cf) do
+          create(:user_wp_custom_field,
+                 name: 'User CF',
+                 types: [type],
+                 projects: [project])
+        end
+        let(:multi_user_cf) do
+          create(:user_wp_custom_field, :multi_user,
+                 name: 'Multi user CF',
+                 types: [type],
+                 projects: [project])
+        end
+
+        let(:default_cf_values) do
+          {
+            boolean_cf.id => true,
+            required_boolean_cf.id => false,
+            list_cf.id => list_cf.custom_options.find_by(value: "B"),
+            required_list_cf.id => required_list_cf.custom_options.find_by(value: "B"),
+            multi_list_cf.id => multi_list_cf.custom_options.find_by(value: "B"),
+            user_cf.id => dev,
+            multi_user_cf.id => [dev, mover]
+          }
+        end
+
+        before do
+          [work_package, work_package2].each do |wp|
+            wp.update!(custom_field_values: default_cf_values)
+          end
+
+          refresh
+          wait_for_reload
+        end
+
+        it 'clears the chosen values' do
+          # Required fields should not have a 'none' option
+          expect(page).to have_no_select(required_boolean_cf.name, options: ['none'])
+          expect(page).to have_no_select(required_list_cf.name, options: ['none'])
```

Comment:
These expectations could lead to false positives. The `options` attribute allows you to specify **exact** matches. Meaning it's trying to ensure there is no required select field with _solely_ the 'none' option (which there never will be) even if this behavior later changes to explicitly be that.

This is visible if you try:
```ruby
expect(page).to have_select(required_boolean_cf.name, options: ['none'])
```

you'll see the failure output:
```
     Failure/Error: expect(page).to have_select(required_boolean_cf.name, options: ['none'])
       expected to find visible select box "Required Bool CF" that is not disabled with options ["none"] but there were no matches. Also found "(No change)\nyes\nno", which matched the selector but not all filters. Expected options ["none"] found ["(No change)", "yes", "no"]
```

A more precise matcher for this case would be `with_options` which tests against *partial* matches. Meaning it will assert there are no required select fields with a 'none' option included in their option set.

If we then try:
```ruby
expect(page).to have_select(required_boolean_cf.name, with_options: ['none'])
```

you'll see:
```
     Failure/Error: expect(page).to have_select(required_boolean_cf.name, with_options: ['none'])
       expected to find select box "Required Bool CF" that is not disabled with at least options ["none"] but there were no matches
```

I'd suggest the `with_options` matcher instead to ensure no false positives :)

```suggestion
          expect(page).to have_no_select(required_boolean_cf.name, with_options: ['none'])
          expect(page).to have_no_select(required_list_cf.name, with_options: ['none'])
```